packages/lib/delivery/src/notifications/getEmailTemplate.ts

```diff
@@ -5,11 +5,14 @@ import {
     NEW_MSG_EMAIL_TEMPLATE,
 } from './templates/newMessage';
 import { OTP_EMAIL_SUBJECT, OTP_EMAIL_TEMPLATE } from './templates/otp';
-import { generateOtp } from './generateOtp';
+
+// OTP expiration time in seconds
+const OTP_EXPIRY_DURATION = 600; // 10 minutes
 
 // to fetch subject & template of email based on notification type
 export const fetchEmailSubjectAndTemplate = (
     notificationType: NotificationType,
+    mailContent: string,
```

Comment:
As mentioned above mailContant and deliveryInformation are related values to the corresponding template. Please use an generic values argument that supplies the selected template with all values needed.